search_that_hash/printing.py

```diff
@@ -53,8 +53,8 @@ def sth_print(chash, result, type, verified):
         console.print(texts)
 
     @staticmethod
-    def one_print(chash, result):
+    def one_print(chash, result):  # pragma: no cover
```

Comment:
Is this never called? Delete it lol

Reply:
Yeah, I deleted it in my branch cause it was never called lol

Reply:
@SkeletalDemise can you confirm?

Reply:
It's called when STH doesn't find it but another API does :)